### PR TITLE
initial stab at log level conventions for RackHD

### PR DESCRIPTION
**Pull request by heckj**
per https://github.com/RackHD/RackHD/issues/34

### CODE REVIEW

**Comment by heckj** [comment]
@RackHD/corecommitters - per refined down log levels, I took a stab at documentation for conventions on those levels. Please consider this a strawman - would like your feedback for any recommended changes prior to merging.

File: docs/devguide/log_levels.rst [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+  Information logged at the **debug** is not expected to be actionable, but may
+  be expected to be used in external systems collecting the log information for
+  debugging or post-analysis metrics.
+
+Setting up and using Logging
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Using our dependency injection libraries, it's typical to inject ``Logger`` and
+then use it within appropriate methods. Within factory methods for services or
+modules, ``Logger`` is initialized wiht the module name, which annotates the
```
Comment (benbp) [inline docs/devguide/log_levels.rst]: "wiht"

**Comment by benbp** [comment]
:+1: with minor typo fix.

**Comment by DavidjohnBlodgett** [comment]
:+1:

**Comment by brianparry** [comment]
:+1: Thanks @heckj!

**Pull request merged**